.github/workflows/build.yml

```diff
@@ -46,40 +69,179 @@ jobs:
         run: pre-commit install-hooks
       - name: Run pre-commit on all files
         run: pre-commit run --all-files
-  build:
+
+  prepare:
+    # Calculates and publishes outputs that are used by other jobs.
+    #
+    # Outputs:
+    #   created:
+    #     The current date-time in RFC3339 format.
+    #   repometa:
+    #     The json metadata describing this repository.
+    #   source_version:
+    #     The source version as reported by the `bump_version.sh show` command.
+    #   tags:
+    #     A comma separated list of Docker tags to be applied to the images on
+    #     DockerHub.  The tags will vary depending on:
+    #     - The event that triggered the build.
+    #     - The branch the build is based upon.
+    #     - The git tag the build is based upon.
+    #
+    #     When a build is based on a git tag of the form `v*.*.*` the image will
+    #     be tagged on DockerHub with multiple levels of version specificity.
+    #     For example, a git tag of `v1.2.3+a` will generate Docker tags of
+    #     `:1.2.3_a`, `:1.2.3`, `:1.2`, `:1`, and `:latest`.
+    #
+    #     Builds targeting the default branch will be tagged with `:edge`.
+    #
+    #     Builds from other branches will be tagged with the branch name.
+    #     Solidi `/` in branch names are replaced with hyphens `-` in the Docker
```

Comment:
`Solidi`?  What do [ancient Roman gold coins](https://www.merriam-webster.com/dictionary/solidi) have to do with this?

Reply:
`Solidus` is the proper name for [that Unicode character](https://unicodelookup.com/#solidus/1).   
`hyphen` should technically be [`hyphen-minus`](https://unicodelookup.com/#hyphen-minus/1).    
Good catch!

Reply:
TIL about the `solidus` Unicode character! 👍